support-workers/src/test/scala/com/gu/support/workers/integration/SendThankYouEmailSpec.scala

```diff
@@ -137,6 +138,40 @@ object SendContributionEmail extends App {
   sendSingle(ef)
 
 }
+
+object SendSupporterPlusEmail extends App {
```

Comment:
if you run this locally with AWS creds, it sends an email and also displays all the field names and values on the console so you can check it makes sense.